packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -48,6 +48,48 @@ export function isTreeNodeSchemaClass<
  */
 export type AllowedTypes = readonly LazyItem<TreeNodeSchema>[];
 
+/**
+ * Additional information to provide to a Node Schema.
+ *
+ * @typeParam TCustomMetadata - Custom metadata properties to associate with the Node Schema.
+ * See {@link NodeSchemaMetadata.custom}.
+ *
+ * @public
+ */
+export interface NodeSchemaProps<TCustomMetadata = unknown> {
```

Comment:
Personally, I find APIs more readable if the variance is explicitly noted. Here that would be:

```suggestion
export interface NodeSchemaProps<out TCustomMetadata = unknown> {
```

Reply:
This is a habit I need to develop. I always forget to specify this. Good callout!